### PR TITLE
Update MAINTAINERS.adoc

### PR DESCRIPTION
**Pull request by evanchooly**
missed one

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]

File: MAINTAINERS.adoc [hunk 1]
```diff
@@ -103,6 +103,9 @@ If you think some information is outdated, either provide a pull request or send
 |Hibernate ORM with Panache
 |https://github.com/FroMage[Stéphane Épardaud]
 
+|Hibernate ORM with Panache for Kotlin
+|https://github.com/evanchooy[Justin Lee]
```
Comment (machi1990) [inline MAINTAINERS.adoc:107]:
```suggestion
|https://github.com/evanchooly[Justin Lee]
```

File: MAINTAINERS.adoc [hunk 2]
```diff
@@ -158,7 +161,7 @@ If you think some information is outdated, either provide a pull request or send
 |https://github.com/mariofusco[Mario Fusco], https://github.com/cristianonicolai[Cristiano Nicolai]
 
 |Kotlin
-|https://github.com/geoand[Georgios Andrianakis]
+|https://github.com/geoand[Georgios Andrianakis], https://github.com/evanchooy[Justin Lee]
```
Comment (machi1990) [inline MAINTAINERS.adoc:164]:
```suggestion
|https://github.com/geoand[Georgios Andrianakis], https://github.com/evanchooly[Justin Lee]
```

**Comment by gastaldi** [comment]
@evanchooly this PR needs to be rebased :)

File: MAINTAINERS.adoc [hunk 3]
```diff
@@ -179,7 +182,10 @@ If you think some information is outdated, either provide a pull request or send
 |https://github.com/cescoffier[Clément Escoffier]
 
 |MongoDB with Panache
-|https://github.com/loicmathieu[Loïc Mathieu]
+|https://github.com/loicmathieu[Loïc Mathieu], https://github.com/evanchooy[Justin Lee]
```
Comment (machi1990) [inline MAINTAINERS.adoc:185]:
```suggestion
|https://github.com/loicmathieu[Loïc Mathieu], https://github.com/evanchooly[Justin Lee]
```

File: MAINTAINERS.adoc [hunk 4]
```diff
@@ -179,7 +182,10 @@ If you think some information is outdated, either provide a pull request or send
...
+
+|MongoDB with Panache for Kotlin
+|https://github.com/evanchooy[Justin Lee]
```
Comment (machi1990) [inline MAINTAINERS.adoc:188]:
```suggestion
|https://github.com/evanchooly[Justin Lee]
```

**Review by machi1990 — CHANGES_REQUESTED** [review]

File: MAINTAINERS.adoc (hunk 1 above)
Comment (gastaldi) [reply MAINTAINERS.adoc:107]: Ah good catch, maybe @evanchooly didn't want to be found :smile:

Comment (machi1990) [reply MAINTAINERS.adoc:107]: Ha ha maybe `evanchooy` will know :-P
 
Noticed it in the earlier PR but it was already merged.

**Comment by evanchooly** [comment]
le sigh.  never rush even "simple" changes...

**Comment by gastaldi** [comment]
Thanks, can you squash your commits?

**Comment by evanchooly** [comment]
hrm.  the web UI doesn't allow that.  i'll check this branch out locally i suppose.  un momento.

**Comment by gastaldi** [comment]
I wonder why the CI was triggered for this file since it is listed to be ignored: https://github.com/quarkusio/quarkus/blob/master/.github/workflows/ci-actions.yml#L14

**Pull request closed without merging**